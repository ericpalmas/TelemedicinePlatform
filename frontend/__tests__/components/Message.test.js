import React from 'react'
import renderer from 'react-test-renderer'
import Message from '../../src/components/Message'
import { render, screen, cleanup } from '@testing-library/react'
import 'core-js/stable'
import 'regenerator-runtime/runtime'

afterEach(cleanup)
describe('Testing message component', () => {
  test('message renders correctly', () => {
    const tree = renderer.create(<Message children={'errore'} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
  //   test('footer text renders correctly', async () => {
  //     const { getByTestId } = render(<Message />)
  //     const { getByText } = within(getByTestId('footerText'))
  //     expect(getByText('Copyright Â© MyWebSite')).toBeInTheDocument()
  //   })
  it('renders', () => {
    const { asFragment } = render(<Message children='Errore' />)
    expect(asFragment()).toMatchSnapshot()
  })
})
