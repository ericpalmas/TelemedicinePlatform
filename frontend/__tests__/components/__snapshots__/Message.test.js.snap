// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing message component message renders correctly 1`] = `
<div
  className="fade alert alert-info show"
  role="alert"
>
  errore
</div>
`;

exports[`Testing message component renders 1`] = `
<DocumentFragment>
  <div
    class="fade alert alert-info show"
    role="alert"
  >
    Errore
  </div>
</DocumentFragment>
`;
